airbyte-integrations/connectors/source-google-drive/integration_tests/spec.json

```diff
@@ -0,0 +1,395 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/google-drive",
+  "connectionSpecification": {
+    "title": "Google Drive Source Spec",
+    "description": "Used during spec; allows the developer to configure the cloud provider specific options\nthat are needed when users configure a file-based source.",
+    "type": "object",
+    "properties": {
+      "start_date": {
+        "title": "Start Date",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00.000000Z. Any file modified before this date will not be replicated.",
+        "examples": ["2021-01-01T00:00:00.000000Z"],
+        "format": "date-time",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{6}Z$",
+        "pattern_descriptor": "YYYY-MM-DDTHH:mm:ss.SSSSSSZ",
+        "order": 1,
+        "type": "string"
+      },
+      "streams": {
+        "title": "The list of streams to sync",
+        "description": "Each instance of this configuration defines a <a href=\"https://docs.airbyte.com/cloud/core-concepts#stream\">stream</a>. Use this to define which files belong in the stream, their format, and how they should be parsed and validated. When sending data to warehouse destination such as Snowflake or BigQuery, each stream is a separate table.",
+        "order": 10,
+        "type": "array",
+        "items": {
+          "title": "FileBasedStreamConfig",
+          "type": "object",
+          "properties": {
+            "name": {
+              "title": "Name",
+              "description": "The name of the stream.",
+              "type": "string"
+            },
+            "globs": {
+              "title": "Globs",
+              "description": "The pattern used to specify which files should be selected from the file system. For more information on glob pattern matching look <a href=\"https://en.wikipedia.org/wiki/Glob_(programming)\">here</a>.",
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "legacy_prefix": {
+              "title": "Legacy Prefix",
+              "description": "The path prefix configured in v3 versions of the S3 connector. This option is deprecated in favor of a single glob.",
+              "airbyte_hidden": true,
+              "type": "string"
+            },
```

Comment:
This can be deleted.

Reply:
These are only here because I'm pulling in the config classes as-is from the CDK: https://github.com/airbytehq/airbyte/blob/e3793c1491283bc7584a9d2593f99e00777af2be/airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py#L34

The azure blob storage connector also inherited them: https://github.com/airbytehq/airbyte/blob/31e8170cd2ba1e61ba6f81c60afb5711a652533d/airbyte-integrations/connectors/source-azure-blob-storage/integration_tests/spec.json#L42

I think we should remove them from the CDK and move them to the S3 connector if they are specific for it. For this PR I removed them with specific code, but I will open an issue for the cleanup.